src/Runtime/Distributions/Automata/WeightFunctions.PointMassWeightFunction.cs

```diff
@@ -129,12 +132,12 @@ public PointMassWeightFunction Sum(PointMassWeightFunction weightFunction)
                 throw new NotSupportedException($"{nameof(PointMassWeightFunction)} is not closed under summation.");
             }
 
-            public PointMassWeightFunction Sum(double weight1, double weight2, PointMassWeightFunction weightFunction)
+            public PointMassWeightFunction Sum(double weight1, PointMassWeightFunction weightFunction, double weight2)
```

Comment:
Why change the order of arguments?

Reply:
I didn't put much consideration into that order when I initially defined `IWeightFunction` interface, because I expected end-users to interact with it very rarely if at all. On the other hand `IImmutableDistribution` and related interfaces are much closer to the surface, so I was more careful with `Summable` and decided that the order
```
summand1(this) -> weight1 -> summand2 -> weight2
```
seems more natural than
```
summand1(this) -> weight1 -> weight2 -> summand2
```
(and, unfortunately, I can not start with `weight1`, because this is the `summand1`'s instance method).
Then I changed the corresponding method in `IWeightFunction` to align.